app/Http/Controllers/Wiki/LanguagesController.php

```diff
@@ -203,27 +203,32 @@ public function search($search_text)
             $GLOBALS['i18n_id'],
             $key
         ];
-        $languages = cacheRemember('languages_search', $cache_params, now()->addDay(), function () use ($formatted_search, $limit, $key) {
-            $languages = Language::includeAutonymTranslation()
-                ->includeCurrentTranslation()
-                ->filterableByNameOrAutonym($formatted_search)
```

Comment:
filterableByNameOrAutonym is replaced below, which means this is not included:
``` $query->orWhereRaw('match (autonym.name) against (? IN BOOLEAN MODE)', [$formatted_name]);```
We need to keep the result set identical while refactoring for speed.  Is the above clause handled somewhere else?

Reply:
Yes, it is being handled into the new query. The new query has as goal to filter the languages records and language_translations in the beginning by the `$formatted_name`. The main change is to reduce the scope for the languages and language_translations before we added the next left join clauses. According the above the next left join clauses will be performanced with a smaller languages scope and the time will be decreased.

You can see in the next lines:
https://github.com/faithcomesbyhearing/dbp/blob/8c2297c8218dd397f2d70f0090ac963cf1e71bc7/app/Models/Language/Language.php#L429

https://github.com/faithcomesbyhearing/dbp/blob/8c2297c8218dd397f2d70f0090ac963cf1e71bc7/app/Models/Language/Language.php#L444

Overall, the next method was replaced by the two above lines:
https://github.com/faithcomesbyhearing/dbp/blob/8c2297c8218dd397f2d70f0090ac963cf1e71bc7/app/Models/Language/Language.php#L410